group-generators/helpers/data-providers/mirrorxyz/interface-schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Mirror.xyz",
+  "iconUrl": "",
+  "providerClassName": "MirrorXyzSubgraphProvider",
+  "functions": [
+    {
+      "name": "Get Post Collector",
+      "functionName": "getPostCollectors",
+      "countFunctionName": "getPostCollectorsCount",
+      "description": "Returns post collectors for a given post",
+      "args": [
+        {
+          "name": "Contract Address",
+          "argName": "address",
```

Comment:
Regarding your `QueryMirrorXyzInput` type, argName is `contract` here

Reply:
Thanks!